lib/fauna.rb

```diff
@@ -9,6 +9,7 @@
 # Main namespace for the FaunaDB client.
 module Fauna; end
 
+require 'fauna/_json'
```

Comment:
We already have `Connection` which is also internal and not supposed to be used directly. Let's just handle it in a similar way. I say document `FaunaJson` module as being for internal json parsing, and possibly `:nodoc:` all the methods in it (including `to_hash`, if we don't fix it's accessibility to actually be private).